app/screens/ChatbotScreen.tsx

```diff
@@ -266,19 +275,25 @@ const ChatbotScreen = ({
       {/* Chatbot pages screen */}
       {!isLoading && pages && pageIndex >= 0 && pageIndex < pages.length && (
         <View style={styles.container}>
-          <View style={{flex: 1}}>
-            {/* BACK & PROGRESS */}
-            <View style={styles.progressHeader}>
-              <IconButton color={Colors.Primary} icon="arrow-left" onPress={() => { onBack() }} />
-              {/* <SpacerH />
-              <View style={styles.progressBar}>
-                <ProgressBar color={Colors.Primary} progress={progress} />
-              </View> */}
-            </View>
-            <SpacerV />
+          {/* BACK & PROGRESS */}
+          <View style={styles.progressHeader}>
+            <IconButton color={Colors.Primary} icon="arrow-left" onPress={() => { onBack() }} />
+            {/* <SpacerH />
+            <View style={styles.progressBar}>
+              <ProgressBar color={Colors.Primary} progress={progress} />
+            </View> */}
+          </View>
+          <SpacerV />
+          <ScrollView ref={scrollViewRef} contentContainerStyle={{flex: 1}}>
             {/* HEADER */}
             {pages[pageIndex].header !== undefined && (
-              <H2 text={pages[pageIndex].header ?? ""} />
+              <>
```

Comment:
I don't think the react.Fragment is needed here, because it's if-else and therefore only one top level tag. But whatever 😋